src/Document.php

```diff
@@ -341,6 +341,12 @@ public function post_addItem()
                 'items_id'     => $this->input["items_id"]
             ]);
 
+            if ($this->input["itemtype"] == 'Ticket') {
+                $main_item = new Ticket();
```

Comment:
Notification should probably be raised for any `CommonITILObject` implementations.
```suggestion
            if (is_a($this->input["itemtype"], CommonITILObject::class, true)) {
                $main_item = new $this->input["itemtype"]();
```

@orthagh Should we add default notifications for `Change` and `Problem` too?

Reply:
yes